### PR TITLE
V1.44.0 1 stage

### PR DESCRIPTION
**Pull request by shubham2892**
### Summary
Agent 1.44.0 release

### Testing

New tests cover the changes: 

### Licensing

This contribution is under the terms of the Apache 2.0 License:

### CODE REVIEW

File: packaging/amazon-linux-ami/ecs-init.spec [hunk 1]
```diff
@@ -32,7 +32,7 @@
 %endif
 
 Name:           ecs-init
-Version:        1.43.0
+Version:        1.44.0
 Release:        2%{?dist}
```
Comment (yunhee-l) [inline packaging/amazon-linux-ami/ecs-init.spec:36]: this needs to be 1

File: scripts/changelog/CHANGELOG_MASTER [hunk 2]
```diff
@@ -1,4 +1,9 @@
-1.43.0-2
+1.44.0-1
+Shubham Goyal <shugy@amazon.com>
+2020-08-13T12:39:32-08:00
+Cache Agent version 1.44.0
+
+1.43.0-1
```
Comment (yunhee-l) [inline scripts/changelog/CHANGELOG_MASTER:6]: looks like this needs to be 1.43.0-2

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 # Changelog
 
-## 1.43.0-2
+## 1.44.0-1
+* Cache Agent version 1.44.0
+
```
Comment (mssrivas) [inline CHANGELOG.md:5]: it replaced 1.43.0-2

File: packaging/amazon-linux-ami/ecs-init.spec (hunk 1 above)
Comment (shubham2892) [reply packaging/amazon-linux-ami/ecs-init.spec]: Fixed

File: scripts/changelog/CHANGELOG_MASTER (hunk 2 above)
Comment (shubham2892) [reply scripts/changelog/CHANGELOG_MASTER]: Fixed

File: CHANGELOG.md (hunk 3 above)
Comment (shubham2892) [reply CHANGELOG.md:5]: Fixed

File: packaging/amazon-linux-ami/ecs-init.spec [hunk 4]
```diff
@@ -278,6 +278,10 @@ fi
 %endif
 
 %changelog
+%changelog
```
Comment (yunhee-l) [inline packaging/amazon-linux-ami/ecs-init.spec:281]: is this extra %changelog

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.44.0-1
+* Cache Agent version 1.44.0
```
Comment (yunhee-l) [inline CHANGELOG.md:4]: you need to add some details about teresa's logging driver support feature since it was a change in init as well

Comment (yunhee-l) [reply CHANGELOG.md:4]: for example see the entry for 1.42.0-1.

Comment (shubham2892) [reply CHANGELOG.md:4]: Done

**Pull request merged**